src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/MultiPipelineGraph.tsx

```diff
@@ -39,14 +39,24 @@ export const MultiPipelineGraph = () => {
 
   return (
     <>
-      {Object.entries(groupedRuns).map(([date, runsOnDate]) => (
-        <div className={"pgv-stages__group"} key={date}>
-          <p className="pgv-stages__heading">{date}</p>
-          {runsOnDate.map((run) => (
-            <SingleRun key={run.id} run={run} currentJobPath={currentJobPath} />
-          ))}
+      {Object.keys(groupedRuns).length === 0 ? (
+        <div className="pgv-stages__group">
+            <div className="pgv-stages__heading">No runs found</div>
```

Comment:
According to the glossary `build` should be used:
https://www.jenkins.io/doc/book/glossary/#build-1

```suggestion
            <div className="pgv-stages__heading">No builds found</div>
```